components/alert/alert.css

```diff
@@ -0,0 +1,56 @@
+.alert {
+  display: flex;
+  align-items: center;
+  margin: 0.5rem 1rem;
+  padding: 0.5rem 1rem;
+  border-radius: 5px;
+  width: 100%;
+  border: 1px var(--grey) solid;
```

Comment:
Didn't know that we can provide value to the border property in this way as well.

Reply:
Yes, we can.